e2e/utils/log-utils.ts

```diff
@@ -40,7 +40,7 @@ export function logSuccess(title: string, body?: string) {
  * @returns
  */
 export function stripConsoleColors(log: string): string {
-  return log.replace(
+  return log?.replace(
```

Comment:
log is always string. Calling `replace` on an empty string does nothing.

We don't need this change